### PR TITLE
Adding Browser  Color 

### PR DESCRIPTION
**Pull request by alihusnainarshad**
Adding Browser Color #013153 .  Very dark blue In Mobile View

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
+		<meta name="theme-color" content="#013153">
```
Comment (xaprb) [inline layouts/_default/baseof.html]: Can you explain more about this or describe the purpose, or provide a link for me to read? I am unfamiliar with the meta property you added.

File: static/_headers [hunk 2]
```diff
@@ -3,3 +3,5 @@
  X-XSS-Protection: 1; mode=block
  Referrer-Policy: no-referrer
  X-Content-Type-Options: nosniff
+ Content-Security-Policy: default-src 'none'; object-src 'none'; img-src 'self'; script-src 'self'; style-src 'self'; font-src 'self';
```
Comment (xaprb) [inline static/_headers]: The changes to the _headers file don't appear to be related to the PR and should be omitted, unless I'm mistaken

File: layouts/_default/baseof.html (hunk 1 above)
Comment (alihusnainarshad) [reply layouts/_default/baseof.html]: Kindly Have a look at this page.


https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android

File: static/_headers (hunk 2 above)
Comment (alihusnainarshad) [reply static/_headers]: Yes. _headers file is here mistakenly.

File: layouts/_default/baseof.html (hunk 1 above)
Comment (xaprb) [reply layouts/_default/baseof.html]: Thanks! This is a good idea. But I'd like to make it configurable with a property in the site's config.yaml (or toml). If that item exists, this meta tag would be added, otherwise omitted. Please also take a look at the exampleSite and see where would be the best place to document this. Let me know if I can help!

**Comment by alihusnainarshad** [comment]
**Okay** @xaprb I'm **closing** this PR And will create new... for this feature
